### PR TITLE
Option to allow loading external images from common image sharing websites

### PR DESCRIPTION
**Pull request by pejakm**
Just a "proof of concept". I'm using it 'cause I really like to open an image and be able to save it, directly from FaceSlim. For now, I've included these servers:

 * googleusercontent.com
 * tumblr.com
 * pinimg.com
 * media.giphy.com

because those are that my contacts and pages I follow mostly use.

### CODE REVIEW

File: app/src/main/java/org/indywidualni/fblite/activity/MainActivity.java [hunk 1]
```diff
@@ -103,6 +103,7 @@
     private static final int ID_CONTEXT_MENU_SAVE_IMAGE = 2562617;
     private String mPendingImageUrlToSave = null;
     private static String appDirectoryName;
+    private String imgExt = null;
```
Comment (indywidualny) [inline app/src/main/java/org/indywidualni/fblite/activity/MainActivity.java]: Can be a local variable. Not needed here.

**Comment by indywidualny** [comment]
A nice idea. Let's do this.

**Comment by pejakm** [comment]
Great! :D

**Comment by kamathln** [comment]
Ermm. Can this be an option? I don't want other websites to share tracking info.

**Comment by indywidualny** [comment]
Well. It's a point. However it opens image files only, not websites, and it's disabled by default. We can discuss this.

**Comment by pejakm** [comment]
It's optional. It works only if you enable it in settings.

**Comment by kamathln** [comment]
Pejakm: Thanks thats all I wanted to know. 

Tip: Warn the user about privacy the first time he/she uses the option.

**Pull request merged**